invenio_previewer/config.py

```diff
@@ -39,6 +39,8 @@
     "xml_prismjs",
     "mistune",
     "pdfjs",
+    "video_videojs",
+    "audio_videojs",
     "ipynb",
     "zip",
     "txt",
```

Comment:
Any Unit Tests?

Reply:
There are no specialized tests for the previewers. The framing is already tested. It makes sense since most previewers are JS focused and just imply reuse of a third-party library that is already tested.